static/app/utils/sessions.tsx

```diff
@@ -28,3 +35,62 @@ export function getCrashFreeRate(
     ? null
     : getCrashFreePercent(100 - percent(crashedCount ?? 0, totalCount ?? 0));
 }
+
+export function getCrashFreeSeries(
+  groups: SessionApiResponse['groups'] = [],
+  intervals: SessionApiResponse['intervals'] = [],
+  field: SessionField
+): SeriesDataUnit[] {
```

Comment:
```suggestion
): (SeriesDataUnit | null)[] {
```

shouldn't need to cast that null below to any, but we might want to filter it out?

Reply:
I filtered out nulls with lodash's compact because it handles types better than the native filter in this case.